### DISCUSSION TITLE
how do i change npcs?

### DISCUSSION THREAD

**Discussion by spook389** [Pryaxis/TShock#3020] [Q&A]
i´m playing on a server.pro server, and i already registered myself and logged in, i can do things such as using wormhole potions, however i can´t relocate npcs to a new home or start invasions, is there like a higher admin role or something like that?

**Comment by sgkoishi** [comment]
There is `vip`, `newadmin`, `admin`, `trustedadmin` and `owner`, and the `superadmin` (not suggested).

**Reply by spook389** [reply]
and how do i set those roles?
